server/src/main/resources/file.conf

```diff
@@ -82,11 +82,7 @@ store {
 }
 lock {
```

Comment:
I think  should delete the lock node, which is strongly associated with the store. As for the table name, we should use our default table name just like `undolog`, and do not need to allow user settings.

Reply:
> I think should delete the lock node, which is strongly associated with the store. As for the table name, we should use our default table name just like `undolog`, and do not need to allow user settings.

I will store the row locks in the user's database, beside the undo_log table, and need a lock node to configure this switcher.